documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst

```diff
@@ -297,251 +329,557 @@ the position of the plain you want to cut. These fields are ``u`` and ``v``:
+
+.. note::
+
+  :math:`\phi` is in the range between :math:`-180^{\circ}` and :math:`180^{\circ}`
+
+* :math:`E`
+
+  is the energy transfer as defined in ``line_proj``
+
+
+.. figure:: ../images/sphere_proj_graph.jpg
    :align: center
-   :width: 800px
-   :alt: spherical coordinate system. 
+   :width: 400px
+   :alt: spherical coordinate system.
 
-   Spherical coordinate system used by ``sphere_proj`` in case of orthogonal and non-orthogonal reciprocal lattice. In latter case vectors :math:`e_x,e_z` define the plane where orthogonal to :math:`e_z` x-axis (:math:`r_x` on the picture) of spherical coordinate system lies.
+   Spherical coordinate system used by ``sphere_proj``.
 
 
 .. note::
 
-   A spherical projection does not have any scaling aspect to the
-   |Q| in the same way a ``line_proj`` can define non-unitary
-   vectors as the axes.
+   A spherical projection currently does not have the ability to be
+   rescaled to |Q|.
 
-When it comes to cutting and plotting, we can use a ``sphere_proj`` in exactly
-the same way as we would a ``line_proj`` with one key difference. The binning
-arguments of ``cut`` no longer refer to :math:`h,k,l,E`, but to |Q|,
-:math:`\theta`, :math:`\phi`, :math:`E` coordinates.
+When it comes to cutting and plotting, we can use a ``sphere_proj`` in
+exactly the same way as we would a ``line_proj``, but with one key
+difference. The binning arguments of ``cut`` no longer refer to
+:math:`h,k,l,E`, but to |Q|, :math:`\theta`, :math:`\phi`, :math:`E`.
 
 .. code-block:: matlab
 
    sp_cut = cut(w, sp_proj, Q, theta, phi, e, ...);
 
-The structure of the arguments to cut is still the same (see `Binning arguments`_)
+.. warning::
+
+   The form of the arguments to ``cut`` is still the same (see: `Binning
+   arguments`_), however:
 
-By default a ``sphere_proj`` will define its principal axes :math:`e_z` and :math:`e_x` or rather 
-:math:`r_x` for angular integration (:math:`\theta`, :math:`\phi`) along 
-defined :math:`hkl` directions :math:`[1,0,0] (e_z)` and :math:`[0,1,0] (e_x)`. Using ``sphere_proj`` 
-properties :math:`e_z` and :math:`e_x` spherical coordinate system may be reoriented to any
-reciprocal lattice direction. Naturally, :math:`e_z` and :math:`e_x` vectors can not be parallel. 
-   
-   
-``sphere_proj`` 2D and 1D cuts samples:
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Spherical projection allows you to obtain powder average. Note that binning ranges are specified in 
-target coordinate system. Energy transfer by default is expressed in inverse Angstroms and angles 
-are in degrees.
+   - |Q| runs from :math:`[0, \infty)` (attempting to use a |Q| with a minimum
+     bound less than :math:`0` will result in unexpected behaviour)
+   - :math:`\theta` runs between :math:`[0, 180)`
+   - :math:`\phi` runs between :math:`[-180, 180)`
+   - Attempting to use default binning (``[]``, ``[n]``) will produce
+     erroneous results if source object is in a linear projection
+     (``line_proj``).
```

Comment:
above 2 lines are about line_proj not sphere_proj

Reply:
No, they're about taking a cut using default binning arguments from an `sqw` (the source object) which is in a linear projection (`line_proj`)  when your target cut is in a spherical projection.

Reply:
Having spoken to both Christian and Mo, they think the meaning here is clear.